src/main/java/com/aws/greengrass/lifecyclemanager/ShellRunner.java

```diff
@@ -19,14 +19,7 @@
 import static com.aws.greengrass.ipc.AuthenticationHandler.SERVICE_UNIQUE_ID_KEY;
 import static com.aws.greengrass.util.Utils.isEmpty;
 
-public interface ShellRunner {
-
-    Exec setup(String note, String command, GreengrassService onBehalfOf) throws IOException;
-
-    boolean successful(Exec e, String note, IntConsumer background, GreengrassService onBehalfOf)
-            throws InterruptedException;
-
-    class Default implements ShellRunner {
+public class ShellRunner {
```

Comment:
I don't think this change should be necessary. The ShellRunner should be able to call for a Platform, either static or via injection

Reply:
I'm also considering just move the methods here into `GenericExternalService` because that's the only usage right now, for setting up some Exec options. Not sure what was the original purpose of this interface.